dcpy/connectors/s3.py

```diff
@@ -8,9 +18,83 @@ def client(
     endpoint_url=os.environ["AWS_S3_ENDPOINT"],
 ):
     """Returns a client for AWS S3."""
+    config = Config(read_timeout=120)
     return boto3.client(
         "s3",
         aws_access_key_id=aws_access_key_id,
         aws_secret_access_key=aws_secret_access_key,
         endpoint_url=endpoint_url,
+        config=config,
     )
+
+
+## adapted from data library
+## https://github.com/NYCPlanning/db-data-library/blob/main/library/s3.py#L37
+def upload_file(
+    bucket: str,
+    path: str,
+    key: str,
+    acl: str = "public-read",
+    metadata: dict = {},
```

Comment:
So this is a bizarre Python'ism (thanks Guido) but you'd typically pass None here due to [dangerous-default-values](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html

Reply:
also, if you want to put the context above the fn in the docstring, that'd be 👍

Reply:
Good to know